scripts/prepare_release.sh

```diff
@@ -67,7 +67,10 @@ git tag v${VERSION}
 
 # Updates the pom.xml with the next snapshot version.
 # For example, when releasing 1.5.7, the next snapshot version would be 1.5.8-SNAPSHOT.
-NEXT_SNAPSHOT=${NEXT_VERSION}-SNAPSHOT
```

Comment:
Shouldn't we just have `<post-release-version>` not include `-SNAPSHOT` when passing it in?

Reply:
That's not clear, this still handles both behaviors correctly though.